vme/src/act_movement.cpp

```diff
@@ -226,7 +226,7 @@ int room_move(unit_data *ch,
         return 0;
     }
 
-    for (u = UNIT_CONTAINS(ch); u; u = u->getNext())
+    for (u = ch->getContainedUnits(); u; u = u->getNext())
```

Comment:
This is the inverse of UNIT_IN, so would be nice if getContainedUnits was e.g. unitContains(). But it's less confusing than the other one :)

Reply:
I'm ofc also OK with getUnitIn and getUnitContains()

Reply:
Done